test/bailout/rlexe.xml

```diff
@@ -247,6 +247,12 @@
       <baseline>inline_call_bailout.baseline</baseline>
     </default>
   </test>
+  <test>
+    <default>
+      <files>inline_get_bailout.js</files>
+      <compile-flags>-args summary -endargs</compile-flags>
```

Comment:
Sounds reasonable; updated. Thanks!

---
In reply to: [167041772](https://github.com/Microsoft/ChakraCore/pull/4609#discussion_r167041772) [](ancestors = 167041772)

Reply:
Talked with Louis; he recommended against this approach because it makes the baseline file brittle and incurs ongoing costs.

---
In reply to: [167079113](https://github.com/Microsoft/ChakraCore/pull/4609#discussion_r167079113) [](ancestors = 167079113,167041772)